### ISSUE TITLE
slug() regex cleanup

### ISSUE THREAD

**Issue by aleemb** [cocur/slugify#141]
This

    const LOWERCASE_NUMBERS_DASHES = '/([^A-Za-z0-9]|-)+/';

would work just the same if rewritten as

    const LOWERCASE_NUMBERS_DASHES = '/([^A-Za-z0-9])+/';

or even clearer,

    const LOWERCASE_NUMBERS_DASHES = '/([^A-Za-z0-9]+)/';

The hyphen at the end seems superfluous given that the character class match `[^a-zA-Z0-9]` is already capturing the hyphen.

**Comment by aleemb** [comment]
Sorry for the issue title changes. Meant to do them on my own issue tracker

**Comment by florianeckerstorfer** [comment]
Could you please create PR, that would help me a lot.

**Comment by aleemb** [comment]
@florianeckerstorfer I don't use this repo so haven't got a setup. I simply referenced your regex and character replacements but rolled the rest on my own due to different requirements. Would take me considerably longer to setup the repo and tests. However, since the hyphen is superfluous, it really won't make a difference whether you remove it or leave it in. Just wanted to make a note since I noticed it while cleaning up my regex.
